security/custom_provider.rst

```diff
@@ -158,7 +158,7 @@ Here's an example of how this might look::
                 );
             }
 
-            return $this->loadUserByUsername($user->getUsername());
```

Comment:
The point here is that we want to refresh the user. Thus, we must call the webservice for the currently logged in user. We could write the same logic from `loadUserByUsername()` here again. But the example prefers to not duplicate the logic and just use the existing implementation. So, technically the code was already correct. But maybe we could improve the readability by extracting the common code into a private method which we would then use in both `loadUserByUsername()` and `refreshUser()`.